api/tests/unit/domain/services/session-results-link-service_test.js

```diff
@@ -0,0 +1,32 @@
+
+const { expect, sinon } = require('../../../test-helper');
+const sessionResultsLinkService = require('../../../../lib/domain/services/session-results-link-service');
+const tokenService = require('../../../../lib/domain/services/token-service');
+
+const settings = require('../../../../lib/config');
+
+describe('Unit | Domain | Service | Session Results Link Service', () => {
+
+  describe('#generateResultsLink', () => {
+
+    it('should return a valid download link', () => {
+      // given
+      const sessionId = 12345;
+      const daysBeforeExpiration = 30;
+
+      const tokenServiceStub = sinon.stub(tokenService, 'createCertificationResultsLinkToken');
+
+      const token = 'a_valid_token';
+
+      const expectedLink = `${settings.domain.pixApp + settings.domain.tldOrg}/api/sessions/download-all-results/${token}`;
```

Comment:
je pense qu'ici il faut mettre le lien en dur et pas faire la concatenation sinon on a partiellement le même code dans l'implem et dans le test, c'est comme ci on faisait : 

expect(1 + 2).to.equal(1 + 2) 

au lieu de 

expect( 1 + 2).to.equal(3)